src/core/meshcop/commissioner.cpp

```diff
@@ -322,14 +322,13 @@ otError Commissioner::Stop(bool aResign)
     return error;
 }
 
-otError Commissioner::SendCommissionerSet(void)
+void Commissioner::SendCommissionerSet(void)
```

Comment:
Not directly related to this PR, but we will want to consider the case where COMMISSIONER_SET was not delivered to the Leader for some reason.

Reply:
Shall we keep the the `otError`? I guess we probably would handle the failure (retx maybe)  from the method itself, so may be fine to keep it `void`. What do you think?

Reply:
I think changing to `void` is fine.

Reply:
We can grant only one outstanding `SendCommissionerSet` just like what @jwhui  did for `MGMT_*_SET.req` in #4981 .